src/dashboard/Data/Browser/AttachRowsDialog.react.js

```diff
@@ -0,0 +1,92 @@
+import * as Filters from 'lib/Filters';
+import Modal from 'components/Modal/Modal.react';
+import React from 'react';
+import Field from 'components/Field/Field.react';
+import Label from 'components/Label/Label.react';
+import TextInput from 'components/TextInput/TextInput.react';
+
+const styles = {
+  errorList: {
+    margin: '3px 6px',
+    padding: '3px',
+    color: 'red',
+    fontSize: '13px',
+    lineHeight: '1.4',
+    maxHeight: '300px',
+    overflow: 'auto',
+  },
+};
+
+export default class AttachRowsDialog extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      objectIds: '',
+    };
+
+    this.handleObjectIdsChange = this.handleObjectIdsChange.bind(this);
+    this.handleConfirm = this.handleConfirm.bind(this);
+  }
+
+  handleObjectIdsChange(objectIds) {
+    this.setState({ objectIds });
+  }
+
+  handleConfirm() {
+    this.props.onConfirm(this.state.objectIds);
+  }
+
+  render() {
+    const {
+      relation,
+      onCancel,
+      onConfirm,
+      errors,
+    } = this.props;
+    let errorStatus;
+    if (errors) {
+      const errorList = [];
```

Comment:
I meant since `errorList` is being mutated we should use `let` rather than `const`.